resources/views/employees/index.blade.php

```diff
@@ -0,0 +1,151 @@
+                    <th>Email</th>
+                    <th>Phone</th>
+                    <th>Company</th>
+                    <th>Actions</th>
+                </tr>
+            </thead>
+        </table>
+    </div>
+</div><!-- .card-preview -->
+
+@endsection
+
+@push('scripts')
+<script>
+$(document).ready(function() {
+        $('#employee_table').DataTable({
+            processing: true,
+            serverSide: true,
+            ajax: "{{ route('employees.datatable') }}",
+            "columns": [
+                {
+                    name: 'first_name',
+                    data: 'first_name'
+                },
+                {
+                    name: 'last_name',
+                    data: 'last_name'
+                },
+                {
+                    name: 'email',
+                    data: 'email'
+                },
+                {
+                    name: 'phone',
+                    data: 'phone'
+                },
+                {
+                    name: 'company',
+                    data: 'company'
+                },
+                {
+                    name: 'action',
+                    data: 'action',
+                    orderable: false,
+                    searchable: false
+                }
+            ]
+        });
+        $('#employee_table_filter').css('padding-left', '63%');
+        // To Open Create Form
+        $('#add_employee').on('click', function (){
+            axios.get("{{ route('employees.create') }}").then(function (response){
+                $('#modalForm').html(response.data)
+                $('#modalForm').modal('show');
+            }).catch(function (error){
+
+            })
+        })
+        //To Open Edit Form
+        $(document).on('click', '.employee-edit', function (){
+            var url = $(this).attr('edit-url');
+            axios.get(url).then(function (response){
+                $('#modalForm').html(response.data)
+                $('#modalForm').modal('show');
+            }).catch(function (error){
+
+            })
+        })
+        //To Open Delete Confirmation Box & then delete
+        $(document).on('click', '.eg-swal-av3', function(e) {
+            var deleteUrl = $(this).attr('delete-url');
+            Swal.fire({
+                title: 'Are you sure?',
+                text: "You won't be able to revert this!",
+                icon: 'warning',
+                showCancelButton: true,
+                confirmButtonText: 'Yes, delete it!'
+            }).then(function(result) {
+                console.log(result);
+                if (result.value) {
+                    axios.delete(deleteUrl)
+                    .then(function (response){
+                        NioApp.Toast(response.data.message, 'success');
+                        $('#employee_table').DataTable().ajax.reload();
+                    }).catch(function (error){
+                        NioApp.Toast('Unable to delete company', 'error');
```

Comment:
Why is this constant here?

Reply:
Put it for testing purposes. Removed & replaced dynamic string.